src/LinuxTracing/UprobesUnwindingVisitor.h

```diff
@@ -49,14 +49,17 @@ class UprobesUnwindingVisitor : public PerfEventVisitor {
       TracerListener* listener, UprobesFunctionCallManager* function_call_manager,
       UprobesReturnAddressManager* uprobes_return_address_manager, LibunwindstackMaps* initial_maps,
       LibunwindstackUnwinder* unwinder, LeafFunctionCallManager* leaf_function_call_manager,
-      UserSpaceInstrumentationAddresses* user_space_instrumentation_addresses)
+      UserSpaceInstrumentationAddresses* user_space_instrumentation_addresses,
+      const std::map<uint64_t, uint64_t>* absolute_address_to_size_of_functions_to_stop_at = nullptr)
```

Comment:
I think I would prefer to avoid the default parameter. Just pass `nullptr` when you don't have it, I think the same is done with `user_space_instrumentation_addresses`.

Reply:
Ok then.